src/ui/Windows/I18N/Create_DLLs_64.vbs

```diff
@@ -0,0 +1,251 @@
+'Update the 64-bit Resource Only DLLs for each language we support
+
+Option Explicit
+
+If Instr(1, WScript.FullName, "cscript.exe", vbTextCompare) = 0 then
+    MsgBox "This script must be executed by cscript.exe" & vbCRLF & _
+           "e.g. cscript Create_DLLs_64.vbs", _
+           vbCritical, _
+           "Error: " & Wscript.ScriptFullName
+    ' return error code to caller
+    Wscript.Quit(99)
+End If
+
+Dim CURPATH, TOOLS, RESTEXT, RESPWSL, BASE_DLL, DEST_DIR, DO_ALL, DO_COUNTRY
+Dim objFSO
+
+' 64-bit tools
+TOOLS = "..\..\..\..\build\bin"
+RESTEXT = TOOLS & "\restext\release64\ResText.exe"
+RESPWSL = TOOLS & "\respwsl\release64\ResPWSL.exe"
+BASE_DLL = "..\..\..\..\build\bin\pwsafe\release64\pwsafe_base.dll"
+DEST_DIR = "..\..\..\..\build\bin\pwsafe\I18N64\"
+
+Set objFSO = CreateObject("Scripting.FileSystemObject")
+
+' Make absolute addresses
+DEST_DIR = objFSO.GetAbsolutePathName(DEST_DIR)
+DEST_DIR = DEST_DIR & "\"
+
+CURPATH = objFSO.GetAbsolutePathName(".")
+CURPATH = CURPATH & "\"
+
+' Check output directory exists and, if not, create it
+CreateFolder(DEST_DIR)
+If (Not objFSO.FileExists(BASE_DLL)) Then
+  ' Check Base DLL exists
+  WScript.Echo "Can't find the Base DLL - pwsafe_base.dll"
+  WScript.Quit(99)
+End If
+
+If (Not objFSO.FileExists(RESTEXT)) Then
+  ' Check required program exists
+  WScript.Echo "Can't find Tool - ResText.exe"
+  WScript.Quit(99)
+End If
+
+If (Not objFSO.FileExists(RESPWSL)) Then
+  ' Check required program exists
+  WScript.Echo "Can't find Tool - ResPWSL.exe"
+  WScript.Quit(99)
+End If
+
+If Wscript.Arguments.Count = 0 Then
+  DO_ALL = True
+Else
+  DIM Arg1
+  Arg1 = UCase(Wscript.Arguments(0))
+ If Arg1 = "/HELP" Or Arg1 = "/H" Or Arg1 = "/?" Then
+   Usage
+   Wscript.Quit(0)
+  Else
+    DO_ALL = False
+    DO_COUNTRY = Arg1
+  End If
+End If
+
+' Sometimes the delete files do not work for permission failures. So...
+WScript.Echo "*** Please ensure that you have deleted ALL DLLs from this script directory:"
+WScript.Echo  " " & CURPATH
+WScript.Echo  " "
+WScript.Echo "*** Please delete any DLLs you are (re)making from the target directory:"
+WScript.Echo  " " & DEST_DIR
+WScript.Echo  " "
+Pause("Press any key to continue.")
+
+If (objFSO.FileExists("foo.dll")) Then
+  'Delete intermediate DLL if still there
+  objFSO.DeleteFile "foo.dll"
+End If
+
+Dim objStdOut, DoneSome
+Set objStdOut = WScript.StdOut
+
+DoneSome = false
+
+' Now do them
+
+If (DO_ALL = True Or DO_COUNTRY = "CZ") Then

```

Comment:
What would it take to make the addition of a new language "transparent" to this script, i.e., not requiring editing it?